### DISCUSSION TITLE
Installer "quit" at the partition settings stage.. now im bricked

### DISCUSSION THREAD

**Discussion by wallace4793** [1000001101000/Debian_on_Buffalo#171] [Q&A]
Hi,

As the title suggest I was following the youtibe tutorial and all was well except when it got to the stage of the partition setting stage the installer quit - I could see the partition info but rather than it being on an active script/installer it was plain text on the screen. I had no choice but to exit and try again but now it sticks at the first step in the install process "setting up the partitioner". How can I recover things?

I can get the installer up but I cant partition the disks, I think im getting out of memory error.

John

**Comment by 1000001101000** [comment]
Good to hear from you!

What model device is this?

**Comment by wallace4793** [comment]
Hi There :-)

Its a linkstation DUO ls-wxl. Sorry I shouldve posted that. Ive updated the original comment with the image :
![Screenshot 2023-02-06 at 20 22 44](https://user-images.githubusercontent.com/47949758/217080393-b010f4c1-b667-4964-aac2-db8e9b97bddb.png)

Its showing the log screen when it quits and says its out of memory I think.

**Comment by 1000001101000** [comment]
Is this using the Bullseye installer from this project?

I'm a bit surprised this would be happening since you can't be the first ls-wxl trying to run the install, especially since I thought that was the device I did the initial testing with. As memory requirements for the debian installer get larger over time it's getting harder for the older devices to even run the installer but I didn't think we were quite to that point. 

You might be able to make another ssh connection and try activating some swap from terminal, that would probably allow it to work if you can also get the installer to start over.

otherwise, for devices that can't run the installer anymore there is a different procedure which allows you to create a disk image on a PC which you can write to a disk and boot the device into Debian that way:
https://github.com/1000001101000/Debian_on_Buffalo/wiki/Alternate-install-method-via-debootstrap-script

**Comment by wallace4793** [comment]
I followed this guide: 
https://www.youtube.com/watch?v=dfMHlLmsSF8
downloaded the uImage files from the debain website and started with the ssh command which started the installer. I assume this is the bullseye - i chose the latest image from the ls-wxl site. 

I did have some issues moving the files over to the boot folder as it was full so I had to move some file out to a different folder.. I thought this was a risk but moving them to a folder assumed I could copy back if I needed to, im not sure that is related though.

John

**Reply by wallace4793** [reply]
Ill give the bootstrap method a go.... my buffalo has 2 drives I think.. (im not running a raid). which one do i use?

John

**Reply by 1000001101000** [reply]
Assuming you have your data backed up.... wipe both drives then write the image from the debootstrap script to one of them, it should be able to boot like that. The debootstrap page includes some notes on how to extend the system raid arrays to additional disks after booting the first time. 

The installer from my github rather than Debian's site might work. Your log makes it look like it it was loading LVM stuff right before running out of memory, my images exclude LVM to keep them smaller which likely has the side-effect of letting it succeed on devices with 128MB memory like this one.

**Comment by wallace4793** [comment]
[Thanks](url) again, I hadnt seen the seperate installer files in your git - I assumed I was using yours. I can certainly give it a go with your installer... I ca no longer access /boot from the shell when I ssh in from installer@ - can you tell me where to put the new installer files to start the process? 
![Screenshot 2023-02-07 at 10 02 19](https://user-images.githubusercontent.com/47949758/217215254-0fbe601b-9299-4b23-964d-81f83635f8ee.png)

**Comment by 1000001101000** [comment]
you would have to try to mount sda1/md0 as /boot to put any files into it. That will probably be difficult with an installer that isn't fully working but it might be possible.

**Comment by wallace4793** [comment]
So better to start from scratch and re partition with a ext3 boot
partition?

**Comment by 1000001101000** [comment]
yeah, or just connect a drive to a PC as is and load the files that way.

**Comment by wallace4793** [comment]
Hi,

So I have one of the drive connected to my linux installation (moodeaudio). How do I format and partition it correctly? I can see it as sda6 when I cjheck the drives and shows correct size at 1tb.

John

**Reply by 1000001101000** [reply]
I’ve lost track of what you are trying to accomplish. Most likely you want to mount sda1 and boot the *.buffalo files in place to boot into the installer

**Reply by wallace4793** [reply]
I ran out of memory mid install and now I cant boot. Im trying to manually place the installer files onto the drive. Ive now got the drive on a seperate linux install to prepare it for the install but not sure how to partition the drive.

**Comment by DavidDah** [comment]
If You haven't already managed to successefully install Debian I suggest You take look at my guide for terastation,
also installer for new debian version bookworm seems to be easier to use then older debian versions.

https://github.com/DavidDah/Debian_on_Buffalo/wiki/How-to-install-Debian-on-TeraStation-PRO-II-TS-HTGL-R5
